boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -48,16 +52,24 @@ class FsmStateMachineTest : public testing::Test
             get_current_ms, HEARTBEAT_MONITOR_TIMEOUT_PERIOD_MS,
             HEARTBEAT_MONITOR_BOARDS_TO_CHECK, heartbeat_timeout_callback);
 
-        primary_flow_meter = App_FlowMeter_Create(get_primary_flow_rate);
-
-        secondary_flow_meter = App_FlowMeter_Create(get_secondary_flow_rate);
+        primary_flow_meter_in_range_check =
+            App_InRangeCheck_Create(get_primary_flow_rate, 1.0f, 30.0f);
 
+        secondary_flow_meter_in_range_check =
+            App_InRangeCheck_Create(get_secondary_flow_rate, 1.0f, 30.0f);
+        left_wheel_speed_sensor_in_range_check =
+            App_InRangeCheck_Create(get_left_wheel_speed, 0.1f, 150.0f);
+        right_wheel_speed_sensor_in_range_check =
+            App_InRangeCheck_Create(get_right_wheel_speed, 0.1f, 150.0f);
```

Comment:
nit; space for consistency
```suggestion
        secondary_flow_meter_in_range_check =
            App_InRangeCheck_Create(get_secondary_flow_rate, 1.0f, 30.0f);
            
        left_wheel_speed_sensor_in_range_check =
            App_InRangeCheck_Create(get_left_wheel_speed, 0.1f, 150.0f);
            
        right_wheel_speed_sensor_in_range_check =
            App_InRangeCheck_Create(get_right_wheel_speed, 0.1f, 150.0f);
```